samples/switchless/enclave/switchless.conf

```diff
@@ -4,7 +4,7 @@
 # Enclave settings:
 Debug=1
 NumHeapPages=1024
-NumStackPages=1024
-NumTCS=1
+NumStackPages=64
```

Comment:
Just out of curiosity, is it necessary for an enclave to specify both OE_SET_ENCLAVE_SGX and provide a conf file at signing time? Is just one or the other allowed?

Reply:
Enclaves ideally should only provide a conf.The primary use of OE_SET_ENCLAVE_SGX is for tests where we have alternated between not signing and signing test enclaves.

If both are specified, the specifications in conf wins.